Add tokens component docs

blackbaud-johnly opened the pull request:
Addresses https://github.com/blackbaud/skyux2/issues/1849 and https://github.com/blackbaud/skyux2/issues/1848.

blackbaud-conorwright commented on src/app/components/tokens/index.html:
```diff
@@ -0,0 +1,128 @@
+    <sky-demo-page-property
+      propertyName="messageStream"
+    >
+      Specifies ...
```
This is an observable of SkyTokenMessageTypes that can send certain commands to the tokens component. The options are multiple focusing options and one to remove the active token.
Options are: FocusLastToken, FocusActiveToken, FocusPreviousToken, FocusNextToken, and RemoveActiveToken.

blackbaud-conorwright commented on src/app/components/tokens/index.html:
```diff
@@ -0,0 +1,128 @@
+    <sky-demo-page-property
+      propertyName="focusIndexOverRange"
+    >
+      Fires when ...
```
Fires when the active token navigation tries to move past the final token in the list

blackbaud-conorwright commented on src/app/components/tokens/index.html:
```diff
@@ -0,0 +1,128 @@
+      Fires when ...
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="focusIndexUnderRange"
```
Fires when the active token navigation tries to move before the first token in the list

blackbaud-johnly commented:
@blackbaud-conorwright: I updated the PR to incorporate all of your feedback. Thanks again! ... @Blackbaud-SteveBrush: I removed the "HOLD" from the PR title and refreshed it with the latest changes from `master`. Can you please review the updated PR since Conor took the first pass through it? Thanks in advance.

Blackbaud-SteveBrush commented on src/app/components/tokens/index.html:
```diff
@@ -0,0 +1,156 @@
+    <sky-demo-page-property
+      propertyName="focusIndexOverRange"
+    >
+      Fires when users navigate through the token list with the Tab key and attempt to move past the final token in the list.
```
Sometimes it reads "tokens list" and sometimes it reads "token list". Should they be consistent?

blackbaud-johnly replied:
It should be consistent. Thanks for catching that. I'll update all instances of "token list" to be "tokens list."

blackbaud-johnly commented on src/app/components/tokens/index.html:
```diff
@@ -0,0 +1,156 @@
+
+  <sky-demo-page-summary>
+    <p>
+      The tokens component displays a series of specified objects for users to interact with. Users can navigate through the list of tokens and remove tokens from the list. The tokens component's includes a message stream to tokens to the list.
```
This should have been "The tokens component includes a message stream to place focus on tokens or remove tokens from the list."

blackbaud-johnly commented:
@Blackbaud-SteveBrush, I updated the PR to address your feedback. Can you please approve when you get a chance? Thanks!

blackbaud-johnly commented:
@Blackbaud-SteveBrush. does the updated PR look good to you?
